tests/integration/components/user-status-test.js

```diff
@@ -0,0 +1,69 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | user-status', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('show relevant data when status changed to ONBOARDING', async function (assert) {
```

Comment:
`show relevant data when status is ONBOARDING`. Since user doesn't have the flexibility to change to onboarding themselves.